make.js

```diff
@@ -98,9 +83,9 @@ else {
 // for example: ts OR ts,ps
 //
 // note, currently the ts runner igores this setting and will always run.
-process.env['TASK_TEST_RUNNER'] = options.runner || '';
+process.env['TASK_TEST_RUNNER'] = argv.runner || '';
 
-target.clean = function () {
+CLI.clean = function(/** @type {{ suite: string; node: string; … }} */ argv) {
```

Comment:
```suggestion
CLI.clean = function() {
```
argv not used in this function. Should we have it as param then?